src/Address.php

```diff
@@ -12,6 +12,7 @@ class Address
 {
     const HRP = 'erd';
     const CONTRACT_HEX_PUBKEY_PREFIX = '0000000000000000';
+    const HEX_LENGTH = 62;
```

Comment:
I think this should be called: `BECH32_ADDRESS_LENGTH`.

Reply:
True, renamed